app/badges/[badgeSlug]/feedback/page.tsx

```
@@ -0,0 +1,317 @@
+  }
+
+  if (!badge) {
+    return null;
+  }
+
+  if (!content) {
+    return (
+      <div className="page">
+        <aside className="sidebar">
+          <div className={`${styles.sidebarProfile} profile`}>
+            <div className={`${styles.sidebarAvatar} avatar`}>{initialsFromName(displayName)}</div>
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)